test/tests/functional/pbs_cray_smoketest.py

```diff
@@ -130,3 +166,24 @@ def test_cray_compute_job(self):
             # Cray simulator will not show anything in 'use' because
             # aprun command is just a pass through on simulator
             self.assertEqual(cu.count_node_in_state('use'), 1)
+        # verify the contents of output/error files
+        self.server.expect(JOB, {'job_state': 'F'}, id=jid1, extend='x')
+        # sleep a second for the files to be copied
+        time.sleep(1)
```

Comment:
If the job is in state F, its files are already copied.  The job is in state E while the files are being copied.

Reply:
ahhh... you are right! I'll have to change that :)